extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -315,11 +315,22 @@
   flex-direction: row-reverse !important;
 }
 
+.i-amphtml-animate-progress li {
+  transition: transform 0.8s cubic-bezier(0.4, 0.0, 0.2, 1) !important;
+}
+
+.i-amphtml-progress-bar-overflow.i-amphtml-story-progress-bar {
+  width: 100% !important;
+  margin: 4px 0 0 !important;
```

Comment:
Keeping the borders as asked by @newmuis should be as simple as:

```
width: calc(100% - 4px);
margin: 4px 2px 0;
overflow: hidden;
```

Reply:
🤦🏽‍♂️that's easy (i was trying to do it with padding). thanks!

Reply:
Everything got confusing because we all change the box model and set box-sizing to border-box, but margin are outside the content, and padding inside the content :))